connector/mqtt/vernemq/k2v-bridge/README.md

```diff
@@ -1,38 +1,33 @@
-KAFKA_HOSTS              | Address of the kafka broker                                         | kafka-server:9092   | hostname/IP    |
-K2V_MQTT_HOST            | Address of the verne broker                                         | vernemq-k8s         | hostname/IP    |
-K2V_MQTT_PORT            | Port of the verne broker                                            | 8883                | integer        |
-SERVER_HOSTNAME          | DNS of the service to the broker                                    | vernemq-k8s         | hostname       |
-BASE_DIR                 | Base directoty where the project is located                         | /opt/k2v_bridge     | string         |
-K2V_MQTT_USERNAME        | Mqtt client username                                                | k2v-bridge-verne    | string         |
+HOSTNAME                 | Hostname to be used in the certificate common name                  | k2v-bridge          | hostname/IP    |
+EJBCA_ADDRESS            | Address of EJBCA service                                            | ejbca-wrapper:5583  | hostname/IP:port |
+DATA_BROKER_ADDRESS      | Address of the data broker service                                  | data-broker:80      | hostname/IP:port |
+KAFKA_BROKER_LIST        | Addresses of the kafka brokers separated by a comma                 | kafka-server:9092   | hostname/IP:port |
+K2V_LOG_LEVEL            | Log level                                                           | info                | silly, debug, verbose, http, info, warning, error       |
+BASE_DIR                 | Base directory where the project is located                         | /opt/k2v_bridge     | string         |
+K2V_MQTT_USERNAME        | Mqtt client username                                                | k2v-bridge          | string         |
 K2V_MQTT_CLIENT_ID       | Mqtt client client id                                               | hostname            | string         |
-K2V_MQTT_KEEPALIVE       | Mqtt client keepalilve                                              | 60                  | integer        |
-K2V_MQTT_SECURE          | Mqtt client secure                                                  | false               | boolean/string/interger  |
+K2V_MQTT_HOST            | Address of the verne broker                                         | vernemq-k8s         | hostname/IP:port |
+K2V_MQTT_PORT            | Port of the verne broker                                            | 8883                | integer        |
+K2V_MQTT_KEEPALIVE       | Mqtt client keepalive                                               | 60                  | integer        |
+K2V_MQTT_SECURE          | Mqtt client secure                                                  | true                | boolean/string/integer  |
 K2V_MQTT_PUBLISH_TOPIC_SUFFIX   | Mqtt client topic to subscribe                               | /config             | string                   |
```

Comment:
Replace `Mqtt client topic to subscribe` by `Suffix of the mqtt topic to subscribe to`